### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by andrerpena** [adamwdraper/Numeral-js#566]
It's been 6 to 7 months with no commits and several open issues ad PR.

**Comment by perrosnk** [comment]
@andrerpena I am wondering the same thing. Have you found any alternatives to this?

**Comment by perrosnk** [comment]
@adamwdraper ?

**Comment by andrerpena** [comment]
@perrosnk I found no alternative to this and due to the apparent abandonment I ended up implementing the specific functionality I needed with no dependencies.

**Comment by scottie** [comment]
someone wanna make active fork and merge all pull requests ?? get something patched up to 11 months of patches and issues, i will happily help.

**Comment by perrosnk** [comment]
@scottie are you able to do it? I don't know how

**Comment by lutangar** [comment]
There was a discussion of a possible merge with an active fork in #431 but this isn't going to happened. 
Digging even further in #162, there were some people (for example @zeusdeux) interested to maintain this, but... it was 4 years ago!
The best that could happens for this project is that someone willing to take the maintenance come forward and we'll find a way to notify the old maintainer and give him admin rights.

**Comment by BenjaminVanRyseghem** [comment]
See https://github.com/adamwdraper/Numeral-js/issues/242

This project has been dead for years, that's why I forked it and created [numbro](http://numbrojs.com/)

**Comment by zeusdeux** [comment]
@lutangar m still down to maintain this

**Comment by lutangar** [comment]
Hey @adamwdraper , it seems that @zeusdeux is still interested in maintaining this, could you put him as an admin?

**Comment by BenjaminVanRyseghem** [comment]
@lutangar @zeusdeux honestly, I don't see the point in maintaining a project that has been dead for years, with active forks. I think it's a better use of your energy to join efforts where things are moving.

**Comment by zeusdeux** [comment]
@BenjaminVanRyseghem while I agree, the problem is that this repo ranks higher on google and has 4 times the uptake (as per download count on npm) of numbro which is the best fork. Which is why I think investing time here still makes sense.

**Comment by BenjaminVanRyseghem** [comment]
@zeusdeux I agree with you, but I think that more and more people will realize this project is dead and will switch.

@adamwdraper could also add to the README that this project is deprecated/stalled

**Comment by mnemanja** [comment]
So, what's the alternative suggested?

**Comment by BenjaminVanRyseghem** [comment]
@mnemanja I think that people happy with numeral will stay here while people wanting things to move on will gradually migrate to numbro (or another fork)

**Comment by mnemanja** [comment]
@BenjaminVanRyseghem, thanks for the replay.

I've actually chosen the `react-number-format` library which works nicely and is more up to date.

**Comment by JulianKingman** [comment]
d3-format seems pretty well-maintained: https://github.com/d3/d3-format
